synfig-core/src/synfig/string_helper.cpp

```diff
@@ -39,8 +39,16 @@ std::string
 synfig::remove_trailing_zeroes(const std::string& text, bool force_decimal_point)
 {
 	std::string result(text);
+// TODO(ice0): move all the locale related code to the initialization part
+// MinGW C++ std::locale accepts "C" and "POSIX" it does not support other locales.
+#ifdef __MINGW32__
+	struct lconv *locale_info = localeconv();
+	const char decimal_point = *locale_info->decimal_point;
```

Comment:
I think it's better to let it as `char *` as defined in locale_info. `string::find()` works with this parameter type too.

Reply:
Agree.